app/src/main/java/fr/free/nrw/commons/profile/leaderboard/LeaderboardFragment.java

```diff
@@ -172,14 +172,15 @@ private void refreshLeaderboard() {
 
     /**
      * Performs Auto Scroll to the User's Rank
+     * We use userRank+1 to load one extra user and prevent overlapping of my rank button
      */
     private void scrollToUserRank() {
-        if (Objects.requireNonNull(leaderboardListRecyclerView.getAdapter()).getItemCount() > userRank) {
-            leaderboardListRecyclerView.smoothScrollToPosition(userRank);
+        if (Objects.requireNonNull(leaderboardListRecyclerView.getAdapter()).getItemCount() > userRank+1) {
```

Comment:
sorry for the very minor detail, but would you mind making these ` + 1`?
I mean, spaces around the `+` sign, that's the convention in Java :-)

Reply:
Sure!